### PR TITLE
Clarify the settings are appSettings in web.config

### PR DESCRIPTION
**Pull request by stevetemple**
It wasn't entirely clear that these are appSettings in the main web.config.
Also it wasn't clear why you would use which LuceneDirectoryFactory

### CODE REVIEW

**Comment by sofietoft** [comment]
Looks great @stevetemple ! 💪 

I'll make sure that we have this reviewed as soon as possible.

File: Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md [hunk 1]
```diff
@@ -66,15 +67,19 @@ There is a specific documentation for load balancing with [Azure Web Apps](azure
 
 ## Examine Directory Factory Options
  
-- The `TempEnvDirectoryFactory` allows Examine to store indexes directly in the environment temporary storage directory, and should be used instead of `SyncTempEnvDirectoryFactory` mentioned above.
+- The `TempEnvDirectoryFactory` allows Examine to store indexes directly in the environment temporary storage directory, and should be used instead of `SyncTempEnvDirectoryFactory` mentioned below. The `TempEnvDirectoryFactory` should be used on front-end read-only servers.
```
Comment (Jeavon) [inline Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md]: TempEnvDirectoryFactory can be used on back-office servers and often is as it's the option less likely to experience locking issues. I know it's not new but I also think the "should be used instead" should be removed also.

File: Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md [hunk 2]
```diff
@@ -66,15 +67,19 @@ There is a specific documentation for load balancing with [Azure Web Apps](azure
...
+- The `SyncTempEnvDirectoryFactory` enables Examine to sync indexes between the remote file system and the local environment temporary storage directory, the indexes will be accessed from the temporary storage directory. This setting is needed because Lucene has issues when working from a remote file share so the files need to be read/accessed locally. Any time the index is updated, this setting will ensure that both the locally created indexes and the normal indexes are written to. This will ensure that when the app is restarted or the local environment temp files are cleared out that the index files can be restored from the centrally stored index files. This is only required on the server serving the backoffice and not on front-end read-only servers.
```
Comment (Jeavon) [inline Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md]: I think it's difficult to predict a particular system architecture and for this setting to be applied for front-end servers only. In the case of Azure WebApps, this is true, however, we specify that in the Azure specific docs https://github.com/umbraco/UmbracoDocs/blob/master/Getting-Started/Setup/Server-Setup/Load-Balancing/azure-web-apps.md

File: Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md (hunk 1 above)
Comment (stevetemple) [reply Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md]: Maybe the correct approach on this is to just clarify the differences and the benefits of each then. I'll have another look

**Comment by sofietoft** [comment]
Hi @stevetemple !
It's been a while since there's been any activity here.
Did you have a chance to look into the suggestions Jeavon had on the PR? 😄

**Comment by sofietoft** [comment]
Hi @stevetemple !

Did you have a minute or two to take an extra look at this one? 😄

**Comment by stevetemple** [comment]
> Hi @stevetemple !
> 
> Did you have a minute or two to take an extra look at this one? 😄

Yes, I will take another crack at this tomorrow

**Comment by sofietoft** [comment]
Hi there!
I have an important message for you, regarding this PR:

Yesterday we tried to do some cleaning up and force-pushed to the master branch which now causes some problems with existing forks and local clones. 
We have fixed up all the open PRs so that they are still mergeable now.

With that said, if you've been working on a **fork of the repo** or in a **local clone**, those will not work anymore.
If you wish to push any new changes to this PR, you will need to delete your local clone or fork and create fresh ones.

If you have any local work in progress, then you copy it over from your old fork or clone, following these steps:

**Forks**
1. In the old clone of the old fork, switch to the branch you were working on
2. In the new clone of the new fork, create a new branch with the same name as the old one
3. Copy all the affected files from the old clone to the new clone
4. Repeat steps 1-3 if you've been working with multiple branches
5. You can now safely delete your old fork

**Clones**
1. In the old clone, switch to the branch you were working on
2. Create and checkout a new branch on the new clone, using the same name as the old clone
3. Copy all the affected files from the old clone to the new clone
4. Repeat steps 1-3 if you've been working with multiple branches
5. Delete the old clone of the docs repo

And with that, you should be all set once more.

I do apologize for the inconvenience, and please let me know if you have any questions!
/Sofie

**Comment by sofietoft** [comment]
Hi @stevetemple !

To workaround the confusion here, we've decided to [add a tip to the "Examine Directory Factory Options"](https://our.umbraco.com/documentation/Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication#examine-directory-factory-options) section instead, that directs people to the Azure web apps set-up we have.

So, I'll close this one for now!

**Pull request closed without merging**